qiling/extensions/coverage/formats/ezcov.py

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/env python3
+# 
+# Cross Platform and Multi Architecture Advanced Binary Emulation Framework
+#
+
+from os.path import basename
+
+from .base import QlBaseCoverage
+
+
+# Adapted from https://github.com/nccgroup/Cartographer/blob/main/EZCOV.md#coverage-data
+class bb_entry(dict):
```

Comment:
Not sure why this class is inheriting `dict`. Is it being used as a dictionary somewhere else?
If this is only to allow arbitrary fields such as `offset`, `size` and `mod_id`, consider declaring that as a `namedtuple` instead.